generate/generate.go

```diff
@@ -283,13 +284,21 @@ func (g *Generator) requestForPkg(pkgPath string) *plugin.CodeGeneratorRequest {
 	return req
 }
 
+func (g *Generator) addUsedImport(pkgPath string) {
+	if pkgPath == "" {
```

Comment:
when can this happen and why? please add a comment

Reply:
This is just a safe guard for future uses. I will remove it if it is confusing to you though.

Reply:
More than confusing, worrying, Package paths should never be empty, so if we start adding checks for that in the code, we may end up with broken packages that just change gunk's behavior instead of breaking it in obvious ways.